configurations/config.ini.example

```diff
@@ -0,0 +1,7 @@
+[PSQL]
+ENGINE = django.db.backends.postgresql
+NAME =
```

Comment:
What do you think about renaming this variable to DB_NAME? It's a more informative name.